app/models/concerns/import_utils.rb

```diff
@@ -100,6 +94,21 @@ def files_match?(file, url)
     end
   end
 
+  def http_get(url)
+    uri = URI.parse(url)
+    http = Net::HTTP.new(uri.host, uri.port)
+    http.use_ssl = true if uri.scheme == "https"
+    req = Net::HTTP::Get.new(uri)
+    req["User-Agent"] = "PRX-Feeder-Import/1.0 (Rails-#{Rails.env})"
```

Comment:
Moved any HTTP.get requests from the importers into this shared util.  And bonus, I can set a shared user agent string!